pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -45,7 +46,7 @@ import (
 	gcfg "gopkg.in/gcfg.v1"
 	"k8s.io/klog"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
technically, this isn't necessary. the package name is already "v1". but, it doesn't hurt anything I suppose :)

Reply:
Must be my import settings - will clean up.